From df4ccc36b259a39e73dd77c4e87c168a5e3b1040 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@igalia.com>
Date: Thu, 24 Dec 2015 13:51:46 -0600
Subject: [PATCH 03/10] Probably fix icons in search bar

The previous/next icons have been missing for a while, and I finally
decided I cared to fix them. Also, use a better close icon while we're
at it.

This is untested because I cannot figure out how to actually run
empathy-chat in jhbuild. 'jhbuild run empathy' seems to work for the
roster, but start the system version of empathy-chat. Oh well.
---
 libempathy-gtk/empathy-search-bar.ui | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libempathy-gtk/empathy-search-bar.ui b/libempathy-gtk/empathy-search-bar.ui
index cfb8eed..31f6cec 100644
--- a/libempathy-gtk/empathy-search-bar.ui
+++ b/libempathy-gtk/empathy-search-bar.ui
@@ -39,7 +39,7 @@
         <property name="label" translatable="yes">_Previous</property>
         <property name="visible">True</property>
         <property name="is_important">True</property>
-        <property name="icon_name">gtk-go-back-ltr</property>
+        <property name="icon_name">go-previous-symbolic</property>
         <property name="use_underline">True</property>
       </object>
       <packing>
@@ -51,7 +51,7 @@
         <property name="label" translatable="yes">_Next</property>
         <property name="visible">True</property>
         <property name="is_important">True</property>
-        <property name="icon_name">gtk-go-forward-ltr</property>
+        <property name="icon_name">go-next-symbolic</property>
         <property name="use_underline">True</property>
       </object>
       <packing>
@@ -123,7 +123,7 @@
       <object class="GtkToolButton" id="search_close">
         <property name="visible">True</property>
         <property name="is_important">False</property>
-        <property name="stock_id">gtk-close</property>
+        <property name="icon_name">window-close-symbolic</property>
       </object>
       <packing>
         <property name="expand">False</property>
-- 
2.5.0

